### DISCUSSION TITLE
Ionic + React

### DISCUSSION THREAD

**Discussion by hide-username** [devkimc/vite_practice#3] [Q&A] [answered]
- 하이브리드 앱을 위한 Ionic 사용 고려
- react 를 사용하여 앱을 만들 수 있으며, 적은 개발 공수가 필요한 것으로 보임

**Comment by devkimc** [comment, marked as answer]
- react native 와 비교 필요
- cordova 에 대한 이해
- mac, window 환경에서 문제가 있지는 않은지 조사 필요
